vignettes/emodnet.wfs.Rmd

```diff
@@ -1,13 +1,20 @@
 ---
 title: "Accessing and mapping EMODnet data"
 output: rmarkdown::html_vignette
+df_print: "tibble"
```

Comment:
OMG this is so elegant and simple

Reply:
I know right? And there are more options: https://pkgs.rstudio.com/rmarkdown/reference/output_format.html#arguments-1